src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printList(Task[] list, int counter){
+        for(int i = 0; i < counter; i++){
+            System.out.print(i + 1 + ".");
+            printTask(list[i]);
+        }
+    }
+    public static void printTask(Task t){
+       System.out.println("[" + t.getStatusIcon()+ "] " + t.taskName);
+    }
+
+    public static int getTaskNumberArgument(String input){
+        int spaceIndex = input.indexOf(" ");
+        String taskNum = input.substring(spaceIndex + 1);
+        System.out.println(taskNum);
+        return Integer.parseInt(taskNum.trim());
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
Generic names such as "logo" here and possibly elsewhere I suggest being more specific for other users to know exactly what this variable is for